src/odemis/gui/util/__init__.py

```diff
@@ -282,3 +283,18 @@ class AttrDict(dict):
     def __init__(self, *args, **kwargs):
         super(AttrDict, self).__init__(*args, **kwargs)
         self.__dict__ = self
+
+
+@call_in_wx_main
+def enable_tab_on_stage_position(button, stage, pos, target):
+    """
+    Enable the given tab button if the stage is in target position, disable it otherwise
+    :param button: (Button) the Tab button to enable/disable
+    :param stage: (Actuator) the stage driver
+    :param pos: (dict str->float) current position to check its label
+    :param target: (int) target position label (IMAGING, LOADING..etc)
+    """
+    if getCurrentPositionLabel(pos, stage) == target:
+        button.Enable(enable=True)
+    else:
+        button.Disable()
```

Comment:
The point of Enable accepting a boolean is that you can write:
`button.Enable(getCurrentPositionLabel(pos, stage) == target)`
If you find it too obscure, then you can leave the code as it is, but drop the `enable=True` (because the asymmetry brings suspicion)

Reply:
I agree it's asymmetric, it's just button.Enable() alone won't work, needs parameter. 
I changed it as you instructed.